### ISSUE TITLE
Organisation context switch while doing a payment

### ISSUE THREAD

**Issue by maxvisser** [teamforus/me#193]
## steps to reproduce
- [ ] create me identity
- [ ] create 2 organisations
- [ ] apply for fund  with both organisations (on provider dashboard)
- [ ] accept 1, decline the other (as sponsor)
- [ ] do a transaction with me identity
- [ ] fill in amount click on submit
- [ ] error

## Solution
You should only see the organisation that is allowed to scan the QR-code

![image](https://user-images.githubusercontent.com/10818702/49790530-bca43080-fd2e-11e8-9b07-975f90dd9e77.png)

Basically where DanyK is should be the organisation which you were allowed to scan with.

**Comment by maxvisser** [comment]
When you switch organization name does not change on view

**Comment by maxvisser** [comment]
not tested yet

**Comment by maxvisser** [comment]
@GerbenBosschieter can you test this when testing the new product voucher functionality?

**Comment by maxvisser** [comment]
Gerben reproduced issue; its still in Android; iOS is fixed.

**Comment by jamalv** [comment]
@maxvisser can create a video of ios implementation

**Comment by maestrovsmaster** [comment]
![device-2020-04-09-194706](https://user-images.githubusercontent.com/23098862/78928180-d5e6f980-7aa8-11ea-9595-87e03d592961.png)
